src/setup.py

```diff
@@ -205,7 +205,7 @@ def run(self):
         'packaging',
         'paypalrestsdk==1.13.*',
         'phonenumberslite==8.12.*',
-        'Pillow==9.*',
+        'Pillow>=9.1.*',
```

Comment:
I think we should stick with == instead of >= exactly to prevent this sort of problem ;)

Reply:
Ah, that was a remnant of me debugging this issue. Fixed!